File: Entities/Cita.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Entities
{
    public class Cita: BaseEntity
    {
        public int id { get; set;}
        public int id_item { get; set; }
        public int id_cliente { get; set; }
        public int id_empleado { get; set; }
        public DateTime fecha { get; set; }
        public DateTime hora_inicio { get; set; }
        public DateTime hora_fin { get; set; }
        public string codigo { get; set; }

        public int id_sucursal { get; set; }
        public int id_comercio { get; set; }
        public int[] items { get; set; }
    }
}

File: Web_UI/ClientApp/src/app/models/Cita.ts
export class Cita{
    id: number;
    id_item: number;
    id_cliente: number;
    id_empleado: number;
    fecha: Date;
    hora_inicio: Date;
    hora_fin: Date;
    id_sucursal: number;
    items: number[];
    codigo: string;
    hora_inicio_string: string;
    hora_fin_string: string;
}